lerobot-web-be/app/services/teleoperate.py

```
@@ -0,0 +1,206 @@
+                                right_action_map[k[len("right_") :]] = v
+                        left_obs = remap_keys_for_client_and_convert_to_deg(
+                            left_action_map
+                        )
+                        right_obs = remap_keys_for_client_and_convert_to_deg(
+                            right_action_map
+                        )
+                        item = {left_arm_port: left_obs, right_arm_port: right_obs}
+                        self.joint_state_queue.put_nowait(item)
+
+                    else:
+                        robot_arm_port = list(follower_map.values())[0]
```

Ruff (0.14.3)

Prefer `next(iter(follower_map.values()))` over single element slice

Replace with `next(iter(follower_map.values()))`

(RUF015)